content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,310 @@
+distinguish it from other items and products.
 
-If you want to do it or run very advanced traceability, the Odoo
-double-entry management is the feature you need.
+Serial numbers can consist of many different types of characters: they can be strictly numerical,
+they can contain letters and other typographical symbols, or they can be a mix of all of the above.
 
-Configurations
-==============
+The goal of assigning serial numbers to individual products is to make sure that every item's
+history is identifiable when it travels through the supply chain. This can be especially useful for
+manufacturers that provide after-sales services to products that they sell and deliver.
 
-Application configuration
--------------------------
+.. seealso::
+   :doc:`/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots`
 
-First, you need to activate the serial numbers tracking. To do so, go to
-:menuselection:`Configuration --> Settings` in the *Inventory* application. Then,
-enable the feature and hit save.
+Enable lots & serial numbers
+============================
 
-.. image:: serial_numbers/serial_numbers_01.png
-    :align: center
+To track products using serial numbers, the :guilabel:`Lots & Serial Numbers` feature must be
+enabled. To enable this, go to :menuselection:`Inventory app --> Configuration --> Settings`, scroll
+down to the :guilabel:`Traceability` section, and click the box next to :guilabel:`Lots & Serial
+Numbers`. Remember to click the :guilabel:`Save` button to save changes.
 
-Product configuration
----------------------
+.. image:: serial_numbers/serial-numbers-enabled-setting.png
+   :align: center
+   :alt: Enabled lots and serial numbers setting.
 
-Now, you have to configure the products you want to track by serial
-numbers. To do so, go to :menuselection:`Master Data --> Products` and open the product
-of your choice. Edit it and select *Tracking By Unique Serial Number*
-in the *Inventory tab*. Then, click on save.
+Configure serial number tracking on products
+============================================
 
-.. image:: serial_numbers/serial_numbers_02.png
-    :align: center
+Once the :guilabel:`Lots & Serial Numbers` setting has been activated, individual products can now
+be tracked using serial numbers. To configure this, go to :menuselection:`Inventory app --> Products
+--> Products`, and choose a desired product to track.
 
-.. image:: serial_numbers/serial_numbers_03.png
-    :align: center
+Once on the product form, click :guilabel:`Edit`, and click the :guilabel:`Inventory` tab.
 
-Manage Serial Numbers
-=====================
+To track this product using serial numbers, scroll to the :guilabel:`Traceability` section (in the
+:guilabel:`Inventory` tab), select the :guilabel:`By Unique Serial Number` option, and click
+:guilabel:`Save` to save the changes. Existing or new serial numbers can now be selected and
+assigned to newly-received or manufactured batches of this product.
```

Comment:
Can be more succinct here. We've said repeatedly to this point "in order to track using serial numbers" so just say what to do now.

```suggestion
Once on the product form, click :guilabel:`Edit`, navigate to the :guilabel:`Inventory` tab and
scroll to the :guilabel:`Traceability` section. Then, select the :guilabel:`By Unique Serial Number`
option, and :guilabel:`Save` the changes. Existing or new serial numbers can now be selected and
assigned to newly-received or manufactured batches of this product.
```